website/content/docs/job-specification/hcl2/expressions.mdx

```diff
@@ -296,6 +296,10 @@ job "example" {
     ...
 ```
 
+~> **Caveat:** Dynamic blocks are not supported inside blocks that are opaque
```

Comment:
Hello 👋🏻 We recently released the [link rewrite migration ](https://github.com/hashicorp/nomad/pull/15779)and I'm going through open PRs to address any legacy links. All links to other docs should reference the full relative path from developer.hashicorp.com now. Let me know if you have questions!

Reply:
@kendallstrautman should links in the release branches (like `releases/1.2.x`) also use the new link format?